src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig

```diff
@@ -28,45 +28,51 @@
-            </a>
-          </h2>
+    {% if cartView.customerInformation.id %}
+      <p class="mb-0">
+        {{ 'ID:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {{ cartView.customerInformation.id }}
+      </p>
 
-          <p class="mb-0">
-            {{ 'Account registration date:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-            {{ cartView.customerInformation.registration_date }}
-          </p>
+      <p class="mb-0">
+        {{ 'Name:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {% if cartView.customerInformation.gender %}
+          {{ cartView.customerInformation.gender }}
+        {% endif %}
+        {{ cartView.customerInformation.first_name }} {{ cartView.customerInformation.last_name }}
+      </p>
 
-          <p class="mb-0">
-            {{ 'Valid orders placed:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-            {{ cartView.customerInformation.valid_orders_count }}
-          </p>
+      <p class="mb-0">
+        {{ 'Email:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {{ cartView.customerInformation.email }}
+      </p>
 
-          <p class="mb-0">
-            {{ 'Total spent since registration:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-            {{ cartView.customerInformation.total_spent_since_registration }}
-          </p>
-        </div>
+      <p class="mb-0">
+        {{ 'Account registration date:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {{ cartView.customerInformation.registration_date }}
+      </p>
 
-        <div class="col text-right">
-          <a href="mailto:{{ cartView.customerInformation.email }}" class="btn btn-outline-secondary">
-            {{ cartView.customerInformation.email }}
-          </a>
-        </div>
-      {% else %}
-        <div class="col">
-          <h2 class="mb-0">{{ 'Guest not registered'|trans({}, 'Admin.Orderscustomers.Feature') }}</h2>
-        </div>
-      {% endif %}
-    </div>
+      <p class="mb-0">
+        {{ 'Valid orders placed:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {{ cartView.customerInformation.valid_orders_count }}
+      </p>
+
+      <p>
+        {{ 'Total spent since registration:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        {{ cartView.customerInformation.total_spent_since_registration }}
+      </p>
+
+      <a href="{{ path('admin_customers_view', {'customerId': cartView.customerInformation.id }) }}" class="btn btn-primary">
+        <i class="material-icons">remove_red_eye</i>
+        {{ 'View customer'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </a>
+      <a href="mailto:{{ cartView.customerInformation.email }}" class="btn btn-secondary">
+        <i class="material-icons">mail</i>
+        {{ 'Write an email'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </a>
+
+    {% else %}
+      <p class="mb-0">{{ 'Customer information was not provided yet.'|trans({}, 'Admin.Orderscustomers.Feature') }}</h2>
```

Comment:
What does it mean exactly? The customer has not signed in?

Reply:
If you go to shop > click add to cart. Cart created, but no customer info.